lighthouse-core/lib/styles-helpers.js

```diff
@@ -21,25 +21,32 @@
  * or name/value pair.
  *
  * @param {!Array} stylesheets A list of stylesheets used by the page.
- * @param {string=} propName Optional name of the CSS property to filter results
- *     on. If propVal is not specified, all stylesheets that use the property are
+ * @param {string|string[]=} propName Optional name of the CSS property/propertys to filter
+ *     results on. If propVal is not specified, all stylesheets that use the property are
  *     returned. Otherwise, stylesheets that use the propName: propVal are returned.
- * @param {string=} propVal Optional value of the CSS property to filter results on.
+ * @param {string|string[]=} propVal Optional value of the CSS property/propertys to filter
+ *     results on.
  * @return {!Array} A list of stylesheets that use the CSS property.
  */
 function filterStylesheetsByUsage(stylesheets, propName, propVal) {
   if (!propName && !propVal) {
     return [];
   }
-
   // Create deep clone of arrays so multiple calls to filterStylesheetsByUsage
   // don't alter the original artifacts in stylesheets arg.
   const deepClone = stylesheets.map(sheet => Object.assign({}, sheet));
 
   return deepClone.filter(s => {
     s.parsedContent = s.parsedContent.filter(item => {
-      const usedName = item.property.name === propName;
-      const usedVal = item.property.val.indexOf(propVal) === 0; // val should start with needle
+      let usedName = '';
+      let usedVal = '';
+      // Prevent indexOf on null value
+      if (propName) {
+        usedName = propName.indexOf(item.property.name) > -1;
+      }
+      if (propVal) {
+        usedVal = propVal.indexOf(item.property.val) > -1;
```

Comment:
should probably handle string and array types differently here since `propVal=box` would trigger on both `box` and `flexbox` here